front/src/modules/companies/constants/companiesFieldDefinitions.tsx

```diff
@@ -0,0 +1,156 @@
+import type {
+  ViewFieldBooleanMetadata,
+  ViewFieldChipMetadata,
+  ViewFieldDateMetadata,
+  ViewFieldMetadata,
+  ViewFieldMoneyMetadata,
+  ViewFieldNumberMetadata,
+  ViewFieldRelationMetadata,
+  ViewFieldTextMetadata,
+  ViewFieldURLMetadata,
+} from '@/ui/editable-field/types/ViewField';
+import {
+  IconBrandLinkedin,
+  IconBrandX,
+  IconBuildingSkyscraper,
+  IconCalendarEvent,
+  IconLink,
+  IconMap,
+  IconMoneybag,
+  IconTarget,
+  IconUserCircle,
+  IconUsers,
+} from '@/ui/icon/index';
+import { Entity } from '@/ui/input/relation-picker/types/EntityTypeForSelect';
+import type { ColumnDefinition } from '@/ui/table/types/ColumnDefinition';
+
+export const companiesFieldDefinitions: ColumnDefinition<ViewFieldMetadata>[] =
```

Comment:
companiesAvailableColumnDefinitions?

I think we should also add "available" in the naming to make clear it's the full list and remove any confusion with the one coming from the views

Reply:
Done